src/components/search-builder/search-components/select/select.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from '@angular/core';
+import { InfiniteScrollLoadFunction } from '../../../../directives/infinite-scroll';
```

Comment:
It might be good to change this to `../directives/infinite-scroll/index` as ng-packagr has been known to fail sometimes when index.ts is assumed and not specified.

Reply:
Thanks, yes, I keep forgetting what the right formula is for this.